cv_next/app/cv/[cvId]/components/commentSection/commentForm.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+import { addComment } from "@/app/actions/comments/addComment";
+import { useSupabase } from "@/hooks/supabase";
+import { useRef } from "react";
+import { RxPaperPlane } from "react-icons/rx";
+import { mutate } from "swr";
+
+const COMMENT_FIELD_NAME = "comment";
+
+export default function CommentForm({ cv }: { cv: CvModel }) {
+  const formRef = useRef<HTMLFormElement | null>(null);
+  const supabase = useSupabase();
+
+  const formAction = async (formData: FormData) => {
+    const userId = (await supabase.auth.getUser()).data.user?.id;
+    if (!userId) throw new Error("User not found"); // TODO: handle this
```

Comment:
Don't forget to handle before merge: I'm fine with adding logging + redirect to home, for now.